lib/online_order.rb

```diff
@@ -0,0 +1,66 @@
+# require_relative "order.rb"
+require_relative "customer.rb"
+
+class OnlineOrder < Grocery::Order
+  attr_reader  :order, :customer, :status, :id, :products
+
+  # initialize with default status as pending
+  def initialize(id, products, customer, status = "pending")
+    @id = id
+    @products = products
+    @order = super(id, products)
```

Comment:
If you're calling `super(id, products)`, then the `initialize` method for `Order` will handle saving them, and this code doesn't have to.